### PR TITLE
Html web page

### PR DESCRIPTION
**Pull request by jeesupark96**
https://user-images.githubusercontent.com/90468158/144144135-edaa5471-92e3-4911-91c3-9c3afd3be7e3.mp4

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      jelly lover in you life(or yourself!), this is the gift that keeps on
+      giving all year round. We offer three tiers of monthly jelly joy, and each
+      one offers their own perks. Here is a breakdown of each, as well as your
+      monthly price.
+    </p>
+    <table>
+      <thead>
+        <tr>
+          <td></td>
+          <td scope="col"><strong>Basic Plan</strong></td>
```
Comment (BrettAlbright) [inline html-web-page/index.html]: This element and the two below it should be `th` elements, that will take care of the boldness of the text for you as well without a strong tag!

File: html-web-page/index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+          <td>$30</td>
+          <td>$50</td>
+          <td>$60</td>
+        </tr>
+      </tbody>
+    </table>
+    <p></p>
+    <form>
+      <label for="Your Name">
+        <p>Like what you see? Become a part of the JOTMC family</p>
```
Comment (BrettAlbright) [inline html-web-page/index.html]: The p tag should be outside of the form. I would recommend replacing the empty p above.

File: html-web-page/index.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        </tr>
+        <tr>
+          <td>Monthly Pricing</td>
+          <td>$30</td>
+          <td>$50</td>
+          <td>$60</td>
+        </tr>
+      </tbody>
+    </table>
+    <p></p>
```
Comment (BrettAlbright) [inline html-web-page/index.html]: I have done this before as well, but one thing you want to avoid doing is using empty elements for formatting purposes.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Nice work! Just a few updates to go!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**